datahub-web-react/src/app/entity/glossaryTerm/GlossaryTermEntity.tsx

```diff
@@ -45,7 +53,55 @@ export class GlossaryTermEntity implements Entity<GlossaryTerm> {
 
     getEntityName = () => 'Glossary Term';
 
-    renderProfile: (urn: string) => JSX.Element = (_) => <GlossaryTermProfile />;
+    renderProfile = (urn) => {
+        return (
+            <EntityProfile
+                urn={urn}
+                entityType={EntityType.GlossaryTerm}
+                useEntityQuery={useGetGlossaryTermQuery as any}
+                tabs={[
+                    {
+                        name: 'Schema',
+                        component: SchemaTab,
+                        properties: {
+                            editMode: false,
+                        },
+                    },
```

Comment:
one last comment- we should add logic to make this tab not visible in the case there is no schema, similar to here:


https://github.com/linkedin/datahub/blob/master/datahub-web-react/src/app/entity/dataset/DatasetEntity.tsx#L93

Reply:
added